test/CMakeLists.txt

```diff
@@ -145,23 +145,40 @@ endmacro()
 
 set(_regexp_dpcpp_backend_required "(xpu_api/algorithms|xpu_api/functional|xpu_api/numerics)")
 set(_regexp_switch_off_checked_it  "(test/general/header_order_ranges|test/parallel_api/algorithm/alg.sorting/alg.min.max|test/parallel_api/ranges|test/xpu_api/random/device_tests|test/xpu_api/random/interface_tests|test/xpu_api/random/statistics_tests|test/parallel_api/numeric/numeric.ops/transform_scan)")
+
+set(_regexp_test_pstl_offload_algorithms "(test/parallel_api/algorithm|test/parallel_api/memory|test/parallel_api/numeric|test/xpu_api)")
+set(_regexp_test_pstl_offload_headers "(test/pstl_offload)")
```

Comment:
I'm wondering about the naming of these variables: 
`*_algorithms` and `*_headers`.  Would a better names be `*_parallel_api` and `*_pstl_offload`?  

Is there a reason to separate them at all?  It seems the only functional difference may be to run the exclusion filter on one and not the other, but it seems that it wouldn't make a difference to run it on both.

Reply:
I have rewritten this part as well, now we have only one include and one exclude regular expressions here.